model/fn-execution/src/main/proto/beam_fn_api.proto

```diff
@@ -119,6 +120,7 @@ message InstructionRequest {
     ProcessBundleRequest process_bundle = 1001;
     ProcessBundleProgressRequest process_bundle_progress = 1002;
     ProcessBundleSplitRequest process_bundle_split = 1003;
+    FinalizeBundleRequest finalize_bundle = 1004;
```

Comment:
What is it meant that `InstructionRequest` is marked `Stable`? Is there any expectation that modifications here are backwards/forward compatible?

Reply:
Its a hint that this proto is unlikely to change. There are no compatibility guarantees yet since we don't have a full portability API done and implemented.